x-pack/plugins/security_solution/server/lib/detection_engine/signals/source_fields_merging/utils/filter_field_entries.ts

```diff
@@ -17,13 +19,18 @@ import { FieldsType } from '../types';
  * those and don't try to merge those.
  *
  * @param fieldEntries The field entries to filter
+ * @param ignoreFields Array of fields to ignore. If a value starts and ends with "/", such as: "/[_]+/" then the field will be treated as a regular expression.
+ * If you have an object structure to ignore such as "{ a: { b: c: {} } } ", then you need to ignore it as the string "a.b.c"
```

Comment:
For posterity, ES fieldnames can match RE syntax, but in testing I wasn't able to break anything here, so more of a FunFact™ than anything...

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/131921493-eeae2006-78ff-49f6-94b6-1ed337bcd147.png" />
</p>

Reply:
Thanks, I think if you really needed it you could escape them like `/\/test\//` and you could only use RegEx to match those strings. So they're not blocked, they just would very annoyingly have to use reg ex to match reg ex.

Really though, the hope is we never have to use this feature other than extreme emergencies and even with that use case we are hopeful we don't have to use it at all.